### PR TITLE
SYM-4424: Badge component

### PR DESCRIPTION
**Pull request by nolan-symphony**
Screen Shot 2021-07-09 at 14 51 01

Styles PR: https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles/pull/111

https://www.figma.com/file/V0VKw74594EyCKHwqYk6LV/%E2%9D%96-SDS-Core-Component-Library?node-id=4394%3A60463

### CODE REVIEW

File: src/components/badge/Badge.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const prefix = 'tk-badge';
+
+export type BadgeProps = {
```
Comment (symphony-adnane) [inline src/components/badge/Badge.tsx:7]: can you extend div props? (you can check other components that already do that ;)

Comment (nolan-symphony) [reply src/components/badge/Badge.tsx]: done

File: stories/Badge.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+};
+
+export const Default = Template.bind({});
+
+Default.args = {
+  children: 'Some Text',
+  variant: ''
+};
+
+export const Badges: React.FC = () => (
```
Comment (AliciaSymphony) [inline stories/Badge.stories.tsx:17]: What do you think about separating each variant in a different story?

Comment (nolan-symphony) [reply stories/Badge.stories.tsx:17]: As you want I followed the Buttons stories structure

Comment (nolan-symphony) [reply stories/Badge.stories.tsx]: Done

Comment (AliciaSymphony) [reply stories/Badge.stories.tsx]: Yes indeed... the Buttons stories is one of the first stories and may need some refinement 
Thanks for your modifications :)

**Review by AliciaSymphony — APPROVED** [review]

**Review by symphony-jean-michael — APPROVED** [review]
LGTM

**Pull request merged**